image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -125,17 +127,28 @@ object FileMetadataReader {
       }
     }
 
-  def getColorModelInformation(metadata: Metadata): Map[String, String] = {
+  def getColorModelInformation(image: File, metadata: Metadata): Future[Map[String, String]] = {
 
-      val pngDir = metadata.getFirstDirectoryOfType(classOf[PngDirectory])
+    val source = addImage(image)
+
+    val formatter = format(source)("%A")
```

Comment:
Oh dear. We are using GraphicsMagick unfortunately, the solution seems a bit ugly but we should fix it of course.